### PR TITLE
Fixing missing error messages

### PR DESCRIPTION
**Pull request by sgrieve**
As reported in #317 server timeout messages were not being propagated to the user. This pr resolves that by importing a missing exception and fixing another exception which was being passed as a tuple instead of a string.

Users will now see an appropriate error message if the server times out:

screen shot 2018-05-24 at 15 00 48

### CODE REVIEW

File: python/nammu/controller/NammuController.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
 from javax.swing.filechooser import FileNameExtensionFilter
 from javax.swing.text import DefaultCaret
 from pyoracc.atf.common.atffile import AtfFile
-from requests.exceptions import RequestException
+from requests.exceptions import RequestException, ConnectTimeout
```
Comment (raquel-ucl) [inline python/nammu/controller/NammuController.py]: Why is ConnectTimeout needed? This might be leftover from trying other things below.

**Comment by sgrieve** [comment]
The most recent commits resolve the swingutils not found error we were seeing on everyone but my computer.

The reason it didn't fail on my machine is that I have a native install of jython, which contained all of the required dependencies. This was located in `~/.local/`, I can now replicate the error by renaming that folder so that the jar cannot find the swingutils dependency. The reason it was swingutils that was failing is simply that it was imported first.

The error is caused by 2 problems:

1. The maven build did not copy the pip installed packages into the jar from the tmp install directory.
1. jython executed inside a jar does not have `Lib/site-packages/` in its path, so will not find these files. 

The first issue is resolved in the pom file edit (74bbb56) by adding another resource to the pom file and the second issue is resolved by adding the fix reported in  http://bugs.jython.org/issue2143 to `main.py` which adds the `Lib/site-packages` path to the python path so that the pip installed modules can be imported. It is added to `main.py`, as it has to done before any other imports are done.

These fixes will allow a jar to be built using `mvn clean install -DskipTests`. Pytest fails for a connected reason, that since the tests were written, there are new dependencies, such as `six`, which, although installed correctly cannot be found when junit calls pytest, due to similar path issues as described above. I am in the process of fixing this in `src/test/java/uk/ac/ucl/rc/development/oracc/nammu/NammuTest.java` and will push once this is dealt with.

Once the tests pass, we can explore if the build works on travis. 🚀

**Comment by sgrieve** [comment]
I have come up against an error that @raquel-ucl seems to have encountered a while ago: https://github.com/pytest-dev/pytest/issues/1853 I am not sure how to resolve it, the solution might be to pin pytest to versions <3.0 but it'd be nice to get this import fixed so we can support more recent version of pytest.

**Comment by sgrieve** [comment]
The relevant changes from this PR have been cherry picked into #338 so this PR can be closed.

**Pull request closed without merging**